### PR TITLE
Updates to ReadMe based on recent set up of AI proxy locally on WSL

### PR DESCRIPTION
**Pull request by vijayamanohararaj**
The change includes few updates to the ReadMe file.

- Including instructions for docker installation for WSL
- Moving some content about service dependencies to high level description section
- Including details to validate if local environment is working correctly
- Updating the port for local endpoint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,23 +34,21 @@ from happening. Most logging happens at `INFO`, which is the default setting.
...
 
-To run the app locally, use `docker compose up` from the repo root.
-
-This will run a webserver accessible at <http://localhost:5000>.
+* To run the app locally, use `docker compose up` from the repo root.
 
 **Note**: You need to provide the API keys in the `config.txt` file
 before the service runs. See the above "Configuration" section.
 
+This will run a webserver accessible at <http://localhost>.
```
Comment (vijayamanohararaj) [inline README.md:48]: The default configuration in the docker file was for port 80, I noticed something similar when testing as well in my local instance. Please let me know if I'm missing something here.

Comment (davidsbailey) [reply README.md:48]: I can confirm that the localhost port is now 80. @cat5inthecradle or @wilkie does this seem like a regression?

Comment (vijayamanohararaj) [reply README.md:48]: I'm guessing its defined here https://github.com/code-dot-org/aiproxy/blob/7706cf3623eb983216a8d8291621ca0ba14e98fe/Dockerfile#L12 and the port was updated by this commit https://github.com/code-dot-org/aiproxy/commit/ab8a5708c51ed637e39993959bd011f38f774c5a, but I'm not able to track down the corresponding PR by searching for the SHA hash in PRs

File: README.md [hunk 2]
```diff
@@ -27,23 +34,21 @@ from happening. Most logging happens at `INFO`, which is the default setting.
...
+
+* To validate if the local environment is running successfully, run `bin\assessment-test.rb` It should print the response for a test assessment.
```
Comment (davidsbailey) [inline README.md:50]: will this be run from within your linux environment? if so, please use a forward slash

File: README.md (hunk 1 above)
Comment (davidsbailey) [reply README.md:48]: @cat5inthecradle did you commit this directly to the main branch, or did someone secretly migrate this repo to LFS too?

 seems intentional enough and I don't see a reason not to keep it on port 80.

**Review by davidsbailey — APPROVED** [review]
one request below, otherwise LGTM!

File: README.md (hunk 2 above)
Comment (vijayamanohararaj) [reply README.md]: Yes, good catch. Updated.

**Pull request merged**